packages/table/test/columnTests.tsx

```diff
@@ -75,13 +76,16 @@ describe("Column", () => {
         expectCellLoading(columnHeaders[1], CellType.COLUMN_HEADER);
         expectCellLoading(columnHeaders[2], CellType.COLUMN_HEADER, false);
 
-        const col0cells = table.element.queryAll(`.${Classes.columnCellIndexClass(0)}`);
+        const col0CellsClass = classNames(Classes.columnCellIndexClass(0), Classes.TABLE_CELL);
+        const col0cells = table.element.queryAll(col0CellsClass);
```

Comment:
🛑 these tests are not actually passing because this always returns an empty array cuz! 
you're querying for `"a b"`, not `".a.b"`. use `getElementsByClassName("a b")`, or refactor to generate CSS selectors instead of HTML `class` attribute values.

same applies for changes below in this file.